src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.logo__link {
+  padding: 10px 0;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  line-height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  text-decoration: solid 2px black;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 37px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+}
```

Comment:
It should work for every link that is active. If i set this class to other elements - i'll see 
<img width="1437" alt="Screenshot 2023-01-25 at 23 59 01" src="https://user-images.githubusercontent.com/84275496/214701072-1ff45873-4aaf-402e-8ed1-1005a6a5bcc7.png">